bears/swift/TailorBear.py

```diff
@@ -0,0 +1,54 @@
+import json
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.results.Result import Result
+from coalib.settings.Setting import path
+
+
+@linter(executable="tailor",
+        output_format=None,
+        prerequisite_check_command=("tailor", "-v"),
+        prerequisite_check_fail_message="Tailor is not installed. Refer "
+        "https://tailor.sh/ for installation details.")
+class TailorBear:
+    """
+    Perform static analysis on Swift source files using Tailor.
+
+    For more information on the analysis visit <https://tailor.sh/>
+    """
+    LANGUAGES = "Swift"
+
+    severity_map = {"warning": RESULT_SEVERITY.NORMAL,
+                    "error": RESULT_SEVERITY.MAJOR}
+
+    def process_output(self, output, filename, file):
+        output = json.loads(output)
+
+        for item in output["files"][0]["violations"]:
+            if "column" in item["location"].keys():
+                column = item["location"]["column"]
+            else:
+                column = None
+
+            yield Result.from_values(
+                origin="{} ({})".format(self.name, item["rule"]),
+                message=item["message"],
+                file=filename,
+                line=item["location"]["line"],
+                column=column,
+                severity=self.severity_map[item["severity"]])
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         max_line_length: str="81",
+                         tailor_config: path=""):
+        """
+        :param max_line_length: Maximum Line length (in characters).
+        :param tailor_config:   Path to Tailor configuration file.
+        """
+        _format = "--format=json"
+        _line_length = "--max-line-length=" + max_line_length
```

Comment:
also I'd join all args into an args tuple so you don't have to list them all in 53/54 again